packit/cli/propose_downstream.py

```diff
@@ -140,6 +157,25 @@ def propose_downstream(
     )
 
     for branch in branches_to_update:
+        description = None
+        rebase_bug_no = None
+
+        # BUG(Gris Ge): buggy code on accessing jumble package_config data
```

Comment:
I would move `rebase_bug_no` to the top level of config and don't cycle through the jobs when using this via CLI. WDYT?

In service, it's taken care of -- no cycling needed. (And you can define properties both globally and on the job level.)

Reply:
The `rebase_bug_no` is removed in V2 patch.